2020-12-03 07:21:29,886 [main] INFO  tools.TestDFSHAAdmin (TestDFSHAAdmin.java:runTool(436)) - Running: DFSHAAdmin -failover --forcefence nn1 nn2
2020-12-03 07:21:30,050 [main] INFO  ha.NodeFencer (NodeFencer.java:fence(92)) - ====== Beginning Service Fencing Process... ======
2020-12-03 07:21:30,050 [main] INFO  ha.NodeFencer (NodeFencer.java:fence(95)) - Trying method 1/1: org.apache.hadoop.ha.ShellCommandFencer(true)
2020-12-03 07:21:30,078 [main] INFO  ha.ShellCommandFencer (ShellCommandFencer.java:tryFence(98)) - Launched fencing command 'true' with pid 5473
2020-12-03 07:21:30,085 [main] INFO  ha.NodeFencer (NodeFencer.java:fence(99)) - ====== Fencing successful by method org.apache.hadoop.ha.ShellCommandFencer(true) ======
2020-12-03 07:21:30,087 [main] INFO  tools.TestDFSHAAdmin (TestDFSHAAdmin.java:runTool(440)) - Err_output:

Output:
Failover from nn1 to nn2 successful

msx-rc 0
